### PR TITLE
0.9.10-hf1

### PR DESCRIPTION
**Pull request by Scarjit**
# 0.9.10 Hotfix 1

- Sensorconnect
- Factoryinput

### CODE REVIEW

File: deployment/helm-repo/cloud-init/development.yaml [hunk 1]
```diff
@@ -130,7 +130,7 @@ write_files:
       echo "Helm installed!"
 
       echo "Adding helm repo..."
-      if ! helm repo add united-manufacturing-hub https://repo.umh.app/; then
+      if ! helm repo add united-manufacturing-hub https://0-9-10-hf1.united-manufacturing-hub.pages.dev/; then
```
Comment (JeremyTheocharis) [inline deployment/helm-repo/cloud-init/development.yaml:133]: :)

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: deployment/helm-repo/cloud-init/development.yaml (hunk 1 above)
Comment (Scarjit) [reply deployment/helm-repo/cloud-init/development.yaml:133]: It's still in testing (therefor also tagged as WIP).
I can't to draft PR, else CF won't deploy

**Comment by Scarjit** [comment]
image

**Comment by Scarjit** [comment]
See https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/1460

**Pull request closed without merging**